CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # sidekiq_publisher
 
-## (Unreleased)
+## (unreleased; 3.0.0)
```

Comment:
Do we have to change this after we push out the actual release? Just curious why we're labeling this as "unreleased"?

Reply:
I was planning to one final update to this PR to remove that -- good question

Reply:
I may as well do that now!